pkg/okteto/context.go

```diff
@@ -236,6 +236,9 @@ func GetContextStore() *ContextStore {
 
 	if ContextExists() {
```

Comment:
how come if "Context exists" we check if is nil inside? 🤔

Reply:
It can exist the file but be an empty json `{}` which led to panics later on the code

Reply:
nit: it that case i would rather call the `ContextExists` `ContextFileExists`